website/src/pages/forum.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import Head from '@docusaurus/Head';
+import DiscourseFeed from '@site/src/components/discourse';
+
+function Events() {
+  return (
+    <Layout>
+      <Head>
+        <title>dbt Discourse</title>
+        <meta name="description" content="Description here" />
+      </Head>
+      <section className='discourse-forum-page'>
+        <div className='container'>   
+          <h1>dbt Discourse</h1>
+          <DiscourseFeed title='Open topics' category='help' status='unsolved'  link_text='See open topics' />
+          <DiscourseFeed title='Cool Problem Solving' category='show-and-tell' after='2022-06-14' status='unsolved' link_text='See more topics' link_href='https://discourse.getdbt.com/c/show-and-tell/22' />
```

Comment:
I saw the conversation about the `after` attribute with Joel above -- will we want a date like this to be hard-coded here? Or is this just an example?

Reply:
I doubt we'd ever hardcode a date in here, and instead would just use the relative calculation 👍

Reply:
@joellabes That's why I'm wondering about the `after='2022-06-14'` on Cool Problem Solving...

Reply:
Removed the `after` property for the Cool Problem Solving component.